lib/rubocop/cop/rspec/multiple_expectations.rb

```diff
@@ -116,23 +113,40 @@ def find_aggregate_failures(example_node)
             .find { |block_node| aggregate_failures?(block_node, ANYTHING) }
         end
 
-        def find_expectation(node, &block)
-          yield if expect?(node) || aggregate_failures_block?(node)
+        def expectation_count(node, count = 0)
+          # do not search  inside of aggregate_failures blocks
+          return count if aggregate_failures_block?(node)
+
+          if node.if_type? || node.case_type?
+            return expectation_count_max(node, count)
+          end
+
+          expectation_count_aggregate(node, count)
+        end
 
-          # do not search inside of aggregate_failures block
-          return if aggregate_failures_block?(node)
+        def expectation_count_max(node, count)
+          node.each_child_node.map do |child|
+            expectation_count(child, count)
+          end.max
+        end
 
+        def expectation_count_aggregate(node, count)
           node.each_child_node do |child|
-            find_expectation(child, &block)
+            count = if expect?(child) || aggregate_failures_block?(child)
```

Comment:
If `expect?(child)` is `true`, is it possible that `expectation_count` returns (well, adds to the passed `count`) something else than 1?
Can this be replaced with just `count + 1`?

Those are questions, and not suggestions, because I'm uncertain.